docs/testcases/UI/System_Management/2232-2444_NAT_Gateway/2232_4.md

```diff
@@ -15,12 +15,12 @@ Test verifies that it isn't possible to add route to the existing route with res
 | 5 | Store *External IP* for the route added at case [2232_2](2232_2.md) | |
 | 6 | Click **Add Route** button | |
 | 7 | Input `<Server_name_1>` from case [2232_2](2232_2.md) into the ***Server name:*** field |  |
-| 8 | Mark ***Specify IP address*** checkbox | ***IP*** field with resolved IP address appears |
+| 8 | Mark ***Specify IP address*** checkbox | ***IP*** field with resolved IP address appears | |
```

Comment:
Remove extra `|` symbol in the end of the row

Reply:
Done